tasks/queue-workflow/webpack.config.js

```diff
@@ -0,0 +1,43 @@
+const path = require('path');
+// path to module root
+const root = path.resolve(__dirname);
+
+module.exports = {
+  mode: process.env.PRODUCTION ? 'production' : 'development',
+  entry: './index.js',
+  output: {
+    libraryTarget: 'commonjs2',
+    filename: 'index.js',
+    path: path.resolve(__dirname, 'dist'),
+    devtoolModuleFilenameTemplate: (info) => {
+      const relativePath = path.relative(root, info.absoluteResourcePath)
+      return `webpack://${relativePath}`;
+    }
+  },
+  externals: [
+    'aws-sdk',
+    'electron',
+    {'formidable': 'url'}
+  ],
+  module: {
+    rules: [
+      {
+        test: /\.js$/,
+        exclude: /node_modules/,
+        use: [
+          {
+            loader: 'babel-loader',
+            options: {
+              cacheDirectory: true
+            },
+          },
+        ],
+      },
+    ],
+  },
+  devtool: 'inline-source-map',
+  target: 'node',
+  optimization: {
+    nodeEnv: false
+  }
+};
```

Comment:
Going to trust it is as correct as it is in the queue-granules and queue-pdr tasks